src/styles/blocks/card.scss

```diff
@@ -0,0 +1,118 @@
+.card {
+  width: $widthCard;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  background: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--margin-right-bottom {
+    margin-right: $marginRightCard;
```

Comment:
it is good creating variables, but variable for margin is killing a bee with a sledgehammer 🐝 🔨 = 💀 🔨